components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtOSGIUtil.java

```diff
@@ -289,7 +293,8 @@ public static void updatePermissions(String applicationName, ApplicationPermissi
     private static void addPermission(ApplicationPermission[] permissions, Registry tenantGovReg) throws RegistryException {
         for (ApplicationPermission permission : permissions) {
             String permissionValue = permission.getValue();
-            if (permissionValue.substring(0, 1).equals("/")) {         //if permissions are starts with slash remove that
+
+            if (("/").equals(permissionValue.substring(0,1))) {         //if permissions are starts with slash remove that
```

Comment:
can't we use "permissionValue.startsWith("/")" ?
correct the grammar errors of the comments.